internal/controller/apiextensions/claim/reconciler.go

```diff
@@ -48,49 +47,35 @@ import (
 // Error strings.
 const (
-	errGetClaim                   = "cannot get composite resource claim"
-	errGetComposite               = "cannot get referenced composite resource"
-	errDeleteComposite            = "cannot delete referenced composite resource"
-	errDeleteUnbound              = "refusing to delete composite resource that is not bound to this claim"
-	errDeleteCDs                  = "cannot delete connection details"
-	errRemoveFinalizer            = "cannot remove composite resource claim finalizer"
-	errConfigureComposite         = "cannot configure composite resource"
-	errPatchComposite             = "cannot patch composite resource"
-	errFixFieldOwnershipComposite = "cannot fix field ownerships on composite resource"
-	errFixFieldOwnershipClaim     = "cannot fix field ownerships on claim resource"
-	errConfigureClaim             = "cannot configure composite resource claim"
-	errPropagateCDs               = "cannot propagate connection details from composite"
-
-	errUpdateClaimStatus = "cannot update composite resource claim status"
-
-	reconcilePausedMsg = "Reconciliation (including deletion) is paused via the pause annotation"
+	errGetClaim             = "cannot get claim"
+	errGetComposite         = "cannot get bound composite resource"
+	errDeleteComposite      = "cannot delete bound composite resource"
+	errDeleteCDs            = "cannot delete connection details"
+	errRemoveFinalizer      = "cannot remove finalizer from claim"
+	errAddFinalizer         = "cannot add finalizer to claim"
+	errUpgradeManagedFields = "cannot upgrade composite resource's managed fields from client-side to server-side apply"
+	errSync                 = "cannot bind and sync claim with composite resource"
+	errPropagateCDs         = "cannot propagate connection details from composite resource"
+	errUpdateClaimStatus    = "cannot update claim status"
+
+	errFmtUnbound = "refusing to operate on composite resource %q that is not bound to this claim: bound to claim %q"
 )
 
+const reconcilePausedMsg = "Reconciliation (including deletion) is paused via the pause annotation"
+
 // Event reasons.
 const (
-	reasonBind               event.Reason = "BindCompositeResource"
-	reasonDelete             event.Reason = "DeleteCompositeResource"
-	reasonCompositeConfigure event.Reason = "ConfigureCompositeResource"
-	reasonClaimConfigure     event.Reason = "ConfigureClaim"
-	reasonPropagate          event.Reason = "PropagateConnectionSecret"
-	reasonPaused             event.Reason = "ReconciliationPaused"
-)
-
-var (
-	// field manager names used by previous Crossplane versions
-	csaManagerNames = []string{"Go-http-client", "crossplane"}
+	reasonBind                event.Reason = "BindCompositeResource"
+	reasonDelete              event.Reason = "DeleteCompositeResource"
+	reasonCompositeConfigure  event.Reason = "ConfigureCompositeResource"
+	reasonClaimConfigure      event.Reason = "ConfigureClaim"
+	reasonClaimSelectDefaults event.Reason = "SelectClaimDefaults"
```

Comment:
Pushed a commit that cleans up unused reasons, thanks!

I was confused because I don't recall explicitly adding `reasonClaimSelectDefaults`. Then I realized this happened because I started the code from the old CSA implementation:

https://github.com/crossplane/crossplane/blob/13c2d1f42c61d332b065f3b20c1db6e10c773c37/internal/controller/apiextensions/claim/reconciler.go#L80